Makefile

```diff
@@ -65,7 +68,7 @@ lint: kondo cljfmt eastwood
 deploy: check-env
 	lein with-profile -user,-dev,+$(VERSION) deploy clojars
 
-# Usage: PROJECT_VERSION=3.2.1 make install
+# Usage: PROJECT_VERSION=0.1.1 make install
```

Comment:
I think was meant to be a placeholder. :-)

Reply:
I commonly replace it along other ocurrences so that it's handy at all times